pkgs/by-name/wi/wideriver/package.nix

```diff
@@ -0,0 +1,46 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  lib,
+  gcc,
```

Comment:
Remove in tandem with https://github.com/NixOS/nixpkgs/pull/311935#discussion_r1607245223